recipes/hdf5-plugins/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "hdf5-plugins" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nexusformat/HDF5-External-Filter-Plugins/archive/v{{ version }}.tar.gz
+  sha256: 3d0d93cb71e1f384e827fbc9808d72b0984195ab337cca697c66205e7c135b0d
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  run:
+    - {{ pin_subpackage(name|lower + '-bitshuffle', exact=True) }}
+    - {{ pin_subpackage(name|lower + '-bzip2', exact=True) }}
+    - {{ pin_subpackage(name|lower + '-lz4', exact=True) }}
+
+outputs:
+  - name: {{ name|lower + '-bitshuffle' }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake >=3.0.0
```

Comment:
Add `make  # [unix]` to all of these instead of depending on the hosts installation.

Reply:
Done